packages/ariakit/src/tooltip/__examples__/tooltip-with-timeout/test.tsx

```diff
@@ -0,0 +1,65 @@
+import {
+  blur,
+  getByRole,
+  hover,
+  press,
+  render,
+  waitFor,
+} from "ariakit-test-utils";
+import Example from ".";
+
+test("render tooltip", () => {
+  const { baseElement } = render(<Example />);
+  expect(baseElement).toMatchInlineSnapshot(`
+    <body>
+      <div>
+        <button
+          aria-labelledby="r:0"
+          class="button"
+          data-command=""
+          tabindex="0"
+          type="button"
+        >
+          Hover or focus on me and wait for 2 seconds
+        </button>
+      </div>
+      <div
+        id="r:0-portal"
+      >
+        <div>
+          <div
+            class="tooltip"
+            hidden=""
+            id="r:0"
+            role="tooltip"
+            style="display: none;"
+          >
+            Tooltip
+          </div>
+        </div>
+      </div>
+    </body>
+  `);
+});
+
```

Comment:
We're already getting the markup on the default example, so I think we don't need it here. These snapshots can be really annoying when updating the code later (you change a single prop, and then get a lot of failing tests, even though the component still works), so I try to keep them at a minimum.

```suggestion
```